### PR TITLE
Add contributing section to readme

### PR DESCRIPTION
**Pull request by bfirsh**
Signed-off-by: Ben Firshman <ben@firshman.co.uk>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +52,11 @@ Installation and documentation
 
 - Full documentation is available on [Docker's website](http://docs.docker.com/compose/).
 - Hop into #docker-compose on Freenode if you have any questions.
+
+Contributing
+------------
+
+[![Build Status](http://jenkins.dockerproject.com/buildStatus/icon?job=Compose Master)](http://jenkins.dockerproject.com/job/Compose%20Master/)
+
+Want to help build Swarm? Check out our [contributing documentation](https://github.com/docker/compose/blob/master/CONTRIBUTING.md).
```
Comment (dnephin) [inline README.md]: `s/Swarm/Compose/` right?

Comment (bfirsh) [inline README.md]: Oops ta

**Comment by bfirsh** [comment]
ping @aanand @dnephin. A lot of people at the birthday events didn't find the contributing documentation.

**Comment by dnephin** [comment]
LGTM, I think it makes sense to link from the README

**Comment by aanand** [comment]
LGTM

**Pull request merged**